registry/root.go

```diff
@@ -75,7 +77,7 @@ var GCCmd = &cobra.Command{
 			os.Exit(1)
 		}
 
-		err = storage.MarkAndSweep(ctx, driver, registry, dryRun)
+		err = storage.MarkAndSweep(ctx, driver, registry, dryRun, manifestswithouttags)
```

Comment:
Adding a new boolean subsequent to another will get confusing. Let's define an options struct here.